src/main/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformation.java

```diff
@@ -48,6 +58,32 @@ public ReleaseCreationInformation(
         this.pegoutCreationRskTxHash = pegoutCreationRskTxHash;
         this.pegoutBtcTx = pegoutBtcTx;
         this.pegoutConfirmationRskTxHash = pegoutConfirmationRskTxHash;
+
+        this.utxoOutpointValues = decodeUtxoOutpointValues(transactionReceipt);
+    }
+
+    private List<Coin> decodeUtxoOutpointValues(TransactionReceipt transactionReceipt) {
+        return transactionReceipt.getLogInfoList().stream()
+            .filter(this::isPegoutTransactionCreatedLog)
+            .findFirst()
+            .map(LogInfo::getData)
+            .map(this::decodePegoutTransactionEventData)
+            .map(UtxoUtils::decodeOutpointValues)
+            .orElse(Collections.emptyList());
+    }
+
+    private boolean isPegoutTransactionCreatedLog(LogInfo log) {
+        Function pegoutTransactionCreatedEvent = BridgeEvents.PEGOUT_TRANSACTION_CREATED.getEvent();
+        final byte[] pegoutTransactionCreatedSignatureTopic = pegoutTransactionCreatedEvent.encodeSignatureLong();
```

Comment:
What's the difference between `encodeSignatureLong` and `encodeSignature`?

Reply:
`encodeSignature()` returns the first 4 bytes of the encoded signature (Keccak256 hash of the signature). `encodeSignatureLong()` returns the full 32 bytes.

Reply:
`encodeSignatureLong` is the full signature, while `encodeSignature` is the shorter one. As @jeremy-then says.

Reply:
Gotcha! Thanks